src/store/AppContext.jsx

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable react-hooks/rules-of-hooks */
 /* eslint-disable react/prop-types */
 import { createContext, useEffect, useState } from 'react';
-import { useCurrentLocation } from '../hooks/useCurrentLocation';
+import { getCurrentLocation } from '../utils/geolocationUtil';
 import assembleAllWeather from '../weatherFunctions/assembleAllWeather';
-import useDate from '../hooks/useDate';
+import useDate from '../utils/dateUtil';
```

Comment:
dateUtil is unclear. Please rename

geolocationUtil and useDate also should be renamed.